src/main/java/com/marklogic/kafka/connect/source/RowManagerSourceTask.java

```diff
@@ -78,8 +80,8 @@ public List<SourceRecord> poll() throws InterruptedException {
             updateMaxConstraintValue(results, queryHandler);
             return newSourceRecords.isEmpty() ? null : newSourceRecords;
         } catch (Exception ex) {
-            // TODO Will include query here soon
-            logger.error("Unable to poll for source records; cause: " + ex.getMessage(), ex);
+            logger.error("Unable to poll for source records; cause: " + ex.getMessage() +
+                "\nCurrent Query: <" + currentQuery + ">", ex);
```

Comment:
Remove the newline symbol, as that can give log analyzers issues too - i.e. they might see the newline as a new log event and not as a continuation of the current log event. Remove the "<" and ">" symbols too, as we don't use markers like that in any other log message - just log whatever the query is, that'll suffice.